src/redux/thunks/downloadK8sSchema.ts

```diff
@@ -0,0 +1,26 @@
+import {createAsyncThunk} from '@reduxjs/toolkit';
+
+import log from 'loglevel';
+import path from 'path';
+
+import {updateK8sVersion} from '@redux/reducers/appConfig';
+import {downloadSchema} from '@redux/services/k8sVersionService';
+
+import {AppDispatch, RootState} from '@shared/models';
+
+export const downloadK8sSchema = createAsyncThunk<void, string, {dispatch: AppDispatch; state: RootState}>(
```

Comment:
I think this has the logic from the `handleDownloadVersionSchema` method in the `Settings` component.
Could we use this thunk in the Settings component as well to reduce duplicate code?